docs/guides/setup-channel.mdx

```diff
@@ -238,17 +225,45 @@ Alternatively, you can also access quantum processors with REST APIs, enabling y
     * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
     * If you don't save your credentials, you must specify them every time you start a new session.
 
+
     <Admonition type="caution">
-        Account credentials are saved in plain text, so only do so if you are using a trusted device.
+      Account credentials are saved in plain text. To more securely manage your token, it is recommended to use an environment variable; see the [Securely manage your token](#securely-manage) section for more information.
     </Admonition>
 
- 1. Test your setup.  Ensure that you can connect to the service:
+6. Test your setup.  Ensure that you can connect to the service:
 
-   ```python
-    from qiskit_ibm_runtime import QiskitRuntimeService
+```python
+  from qiskit_ibm_runtime import QiskitRuntimeService
 
-    service = QiskitRuntimeService()
-  ```
+  service = QiskitRuntimeService()
+```
+
+<span id="securely-manage"></span>
+## Securely manage your token
+
+To store sensitive information like account credentials more securely, it’s a good practice to use environment variables instead of saving them in plain text. Here’s how to modify your code to load the API token from an environment variable.
```

Comment:
Should this be something like "The following describes how"?

Reply:
got it ca16e6962f24e8c93d6e703dcf3b02ffefec076b